frontend/src/queries/useAccountListQuery.ts

```diff
@@ -64,6 +64,9 @@ const AccountsQuery = gql<AccountsListResponse>`
 
 export const useAccountsListQuery = (variables: AccountListVariables) => {
 	const { data } = useQuery({
+		context: {
+			url: inject<string>('fungyApiUrl'),
```

Comment:
This is not really that type safe, it would be better to just fetch the runtime config directly